CMakeLists.txt

```diff
@@ -54,6 +54,13 @@ add_subdirectory(remote-processor)
 
 add_subdirectory(remote-process)
 
+# todo: Can those valgrind flags be moved in CTestCustom.cmake ?
```

Comment:
I tried rely hard but did not succeed. It must be done before the `include(CTest)` the only thing I can do is to create a repertory ctest and include all ctest related things inside. That seems a good idea, isn't it ?